src/azure-cli/setup.py

```diff
@@ -104,7 +104,7 @@
     'azure-mgmt-monitor~=0.5.2',
     'azure-mgmt-msi~=0.2',
     'azure-mgmt-network~=4.0',
-    'azure-mgmt-netapp~=0.5.0',
+    'azure-mgmt-netapp~=0.6.0',
```

Comment:
You also need to upgrade the packages in requirementsxxx.txt file.

Reply:
ok. I've updated what I hope are the correct files.